crates/routes/src/images.rs

```diff
@@ -40,27 +41,22 @@ pub fn config(
     )
     // This has optional query params: /image/{filename}?format=jpg&thumbnail=256
     .service(web::resource("/pictrs/image/{filename}").route(web::get().to(full_res)))
+    .service(web::resource("/pictrs/image/details/original").route(web::get().to(details)))
```

Comment:
Is it really necessary to add another public route?

Reply:
Yep, I tried it at first without, but it failed.

Reply:
The details endpoint only seems to be used in these two places:
- https://github.com/LemmyNet/lemmy/pull/4704/files#diff-2349b3f031b8f368212f4c9a7356679dff7cf8830f7d075f47f582f871c20673R233
- https://github.com/LemmyNet/lemmy/pull/4704/files#diff-0cd13a4a101c05a54d9ab789fa06c9d48ab61c0e1f5c9b9a178440e9ba3a5ef5R371

In both cases it uses the local pictrs url, so there should be no need to expose this endpoint publicly.

Reply:
You're right, I removed it.